### DISCUSSION TITLE
How to modify error message ?

### DISCUSSION THREAD

**Discussion by Georgestom** [ngneat/transloco#659] [Q&A]
How to modify error message ? if anyone has the solution I am interested

**Comment by shaharkazaz** [comment]
@Georgestom Which error message?

**Reply by Georgestom** [reply]
I want to get the "Missing translation for" console error message to interact with in my pre commit

**Reply by shaharkazaz** [reply]
I'm not sure I fully understand what you need but you can either:
1. Disable the logging of the message via the config.
2. Write a custom missing handler and log whatever you want. see the default handler [here](https://github.com/ngneat/transloco/blob/master/libs/transloco/src/lib/transloco-missing-handler.ts#L21)

**Reply by Georgestom** [reply]
i will try, thanks
